### ISSUE TITLE
pvoutput.get_status lite

### ISSUE THREAD

**Issue by JulianBriggs** [openclimatefix/pvoutput#85]
Provide a pvoutput.get_status without importing big libraries: numpy, pandas, cartopy etc
## Detailed Description
Provide a pvoutput.get_status without importing big libraries: numpy, pandas, cartopy etc

## Context

I struggled to install pvoutput on Ubuntu-20.04 LTS due to cartopy requiring late version of proj which is not available on ubuntu 20.04 LTS
And I want to keep Docker images small for rapid development and orchestration
I prefer to get statuses as list of list of tuples of float
rather than pandas frame of numpy float.

Simplify install as above
keep Docker images small for rapid development and orchestration
## Possible Implementation
Separate pvoutput.get_status code into a separate module. Then import that module into the full module.

**Comment by peterdudfield** [comment]
Yea I think thats a great idea. I really like keeping things light-weight as you say. 

Do you want to have a go at this?

**Comment by peterdudfield** [comment]
I just took out 'cartopy' - same as bug you got above. I think (hope) that this package is not critical. I believe this is used for plotting

**Comment by JackKelly** [comment]
yeah, I agree, `cartopy` is probably just used for plotting in the `pvoutput` code.
